### ISSUE TITLE
Azure - Resource health

### ISSUE THREAD

**Issue by brettplarson** [jnoller/kubernaughty#43]
Maybe this is too inside baseball for your vision of this repo, but i'd like to know how the layman can correlate issues related to minor isolated performance degradations on the Azure platform.

Minor issues affecting a node in an AKS Engine VMSS.
![image](https://user-images.githubusercontent.com/33815371/73886821-ec19d480-482f-11ea-9429-76e85cffb1e1.png)

How we can correlate this to a specific node? Can we do anything or is this life? 
Basically how you can get insights into outages that may be occuring outside of your area of responsibility in a platform.

Thanks!

**Comment by jnoller** [comment]
Honestly - that's a good question I would ask on the main https://github.com/Azure/AKS repository - that's beyond my internal understanding of azure (my background is outside MS)
